setup.py

```diff
@@ -288,12 +273,14 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'nose-timer',
     'parameterized',
     'paramiko',
+    'pydevd',
     'pylint~=2.3.1',  # Ensure the same version as in .travis.yml
     'pysftp',
     'pywinrm',
     'qds-sdk>=1.9.6',
     'rednose',
-    'requests_mock'
+    'requests_mock',
+    'virtualenv',
```

Comment:
? - python 3 has `venv` built in too.

Reply:
The problem is that one of the operators (PythonVirtualenv) uses virtualenv command:
```
def _generate_virtualenv_cmd(self, tmp_dir):
        cmd = ['virtualenv', tmp_dir]
        if self.system_site_packages:
            cmd.append('--system-site-packages')
        if self.python_version is not None:
            cmd.append('--python=python{}'.format(self.python_version))
        return cmd
```
And it allows to use different versions of Python even if they are not officially supported for Airflow itself (so it should support running python 2.7 as well). It was actually an undetected bug so far that virtualenv is needed as setup.py dependency - because in the old CI environment virtualenv was always installed anyway. So if I do not add virtualenv here, the tests will fail.

So it's actually a 'fix'. Do you think it's worth to separate it out and make a separate PR for that :) ?

Reply:
Oh right. It's not in any of the other non dev lists? Should probably be a pr on its own in case we have 1.10.5?

Reply:
Nope. It was missing entirely. I can have a separate PR/Jira issue on that - no problem.